### PR TITLE
Move LPC SD options to configs, add MKS_MINI_DISPLAY to SKR boards

### PR DESCRIPTION
**Pull request by jmdearras**
### Requirements

Add pins for the MKS_MINI_12864 display to the pins files for the SKRV1.1 and SKRV1.3

### Benefits

Adds the MKS_MINI_DISPLAY to the SKR boards

### CODE REVIEW

**Comment by jmdearras** [comment]
Sorry for the mess, I haven't done a pull request in a while!

**Comment by gloomyandy** [comment]
Although I can see the logic to it I'm not sure that forcing folks with this display to use the LCD based SD card makes sense. It is not currently done for other displays that have an inbuilt SD card (like the various reprap displays) and seems a little odd that it is done only for this display. Personally I think it makes more sense to keep the choice of how to handle the SD card in just one place. But if you really want to go this way then you should also consider forcing it for any other display that includes a built in reader. 

Perhaps now is the time to consider moving this sort of choice out of the pins file and into the general config files. So that when enabling the SD card the user can make the choice of which reader to use etc. in the same place, rather than having to go hack the pins file? When I added the original SD card options I wasn't sure on the best way of dealing with it and in retrospect I should probably have done a better job of the config. At the very least I think the settings for the SD card should have some way for the user to put the defines in the standard config files and simply have the default here. So perhaps something like....

``` C
#if !defined(USB_SD_ONBOARD) && !defined(USB_SD_DISABLED)
  #define USB_SD_ONBOARD
#endif
#if !defined(LPC_SD_LCD) && !defined(LPC_SD_ONBOARD)
  #define LPC_SD_ONBOARD
#endif
```
This provides a default that will work on all boards (with or without an LCD SD card reader) and if the user prefers to use the LCD reader they can simply add
#define LPC_SD_LCD
to their configuration file. Obviously the comments in the pins file should be updated to reflect this.

File: Marlin/src/pins/pins_BIGTREE_SKR_V1.3.h [hunk 1]
```diff
@@ -212,11 +212,17 @@
 
 #endif // ULTRA_LCD
 
+#if ENABLED(MKS_MINI_12864)       //assign pins for MKS_MINI_12864
+	#define DOGLCD_CS P1_21
+	#define DOGLCD_A0 P1_22
+	#define USB_SD_ONBOARD        // Provide the onboard SD card to the host as a USB mass storage device
+	#define LPC_SD_LCD            // MKS_MINI_12864 uses SD card on the display
+#else
```
Comment (thinkyhead) [inline Marlin/src/pins/pins_BIGTREE_SKR_V1.3.h]: Tabs are verboten! Please configure your editor to use 2 spaces instead of tab characters.

**Comment by thinkyhead** [comment]
@gloomyandy — I've applied your suggestion, and I agree that it makes sense to move these SD hardware options to a central place. That will of course have to be `Configuration_adv.h` — which is applied ahead of `pins.h`. And, we'll have to see what's the best approach where the defaults differ from board to board. Maybe there are enough example configurations to catch them all.

**Pull request merged**